### DISCUSSION TITLE
5차 데모데이 후기

### DISCUSSION THREAD

**Discussion by zeus6768** [woowacourse-teams/2024-code-zap#721] [이야기해봐요]
굿

**Comment by zeus6768** [comment]
## 안건 정하기
- 데모데이 회고
- 6차 데모데이 TODO 작성하기
- 우테코 프리코스 지원자 대상 홍보 전략 기획

**Comment by zeus6768** [comment]
# 데모데이 회고

**Reply by zeus6768** [reply]
- 당일 배포 아쉬움
- 페어프로그래밍
- 미션 먼저 하자

**Comment by zeus6768** [comment]
# 6차 데모데이 TODO 작성하기

## 1. 프로젝트 요구사항

## 2. 개인 미션

## 3. 프리코스 대상 홍보

**Reply by zeus6768** [reply]
## 팀원들이 도입해야 한다고 생각하는 기능

### 짱수
- 사용자 식별: OAuth든 뭐든 비밀번호 찾기 등이 기능할 수 있게
- 근거: 아이디 및 비밀번호 찾기, 악의적 사용자 식별

### 마위
- extension 스크랩 시 출처 표기

### 헤인
- 신고 기능이든 뭐든
  - 신고 버튼을 누른 리소스를 사용자의 입장에서 보이지 않게 개선
- Admin 계정

### 몰리
- 좋아요 도메인 성능개선

### 켬미
- UI/UX 리모델링: 고정 글을 볼 수 있게

**Reply by zeus6768** [reply]
## 필수 개선 과제
- Like, like cancel 엔드포인트 URI `/login`문제 해결
- 검색중 UI
- 구경가기에서 가만히 있으면 500에러 뜬다. -> 재확인 필요
- 프론트 CD 환경 분리
- 에러 코드 
- 문의하기 페이지 분리
- 언어 정도는 자동으로 만들어주기
- 랜딩페이지 개선
- API 속도 개선

**Reply by jminkkk** [reply]
## 백엔드 필수 개선 과제
- Like, like cancel 엔드포인트 URI /login문제 해결
- 구경가기에서 가만히 있으면 500에러 뜬다. -> 재확인 필요
- 에러 코드
- API 속도 개선(좋아요 개선 포함)
- 인스턴스 스왑 메모리 추가

## 목요일까지 할 일 (10/3)
- 인스턴스 스왑 메모리 추가
- 몰리 코드 리뷰

**Reply by Hain-tain** [reply]
프론트엔드 목요일까지 할 일
- 프론트 CD 환경 분리

**Comment by zeus6768** [comment]
# 우테코 프리코스 지원자 대상 홍보 전략 기획

**Reply by zeus6768** [reply]
- 프리코스 지원자 커뮤니티를 우리가 이용해도 되는지 코치들에게 물어보기
- 6기 프리코스 관련 코드를 업로드하고, 홍보자료로 사용한다. 
- 우리가 프리코스 지원자들의 코드리뷰를 하고 코드잽에 업로드된 관련 코드의 링크를 첨부한다.
